app/services/hyrax/custom_queries/navigators/child_file_sets_navigator.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module CustomQueries
+    module Navigators
+      ##
+      # Navigate from a resource to the child filesets in the resource.
+      #
+      # @see https://github.com/samvera/valkyrie/wiki/Queries#custom-queries
+      # @since 3.4.0
+      class ChildFileSetsNavigator
+        # Define the queries that can be fulfilled by this navigator.
+        def self.queries
+          [:find_child_file_sets, :find_child_file_set_ids]
+        end
+
+        attr_reader :query_service
+
+        def initialize(query_service:)
+          @query_service = query_service
+        end
+
+        ##
+        # Find child filesets of a given resource, and map to Valkyrie Resources
+        #
+        # @param [Valkyrie::Resource] resource
+        #
+        # @return [Array<Valkyrie::Resource>]
+        def find_child_file_sets(resource:)
+          query_service.find_members(resource: resource).select(&:file_set?)
+        end
+        alias find_child_filesets find_child_file_sets
+        deprecation_deprecate find_child_filesets: "use find_child_file_sets instead"
```

Comment:
Do you think the alias and its deprecation could be dropped since this is a new class?  

I'm thinking if someone has been calling the old class and deprecated method then they would fix the method call first since it has a deprecation warning but they would have to deliberately change the class by seeing it in the code and would see the old method was gone.  Also it feels like a smell to add a new class with deprecations in it.